package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "dog_chat",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "start": "parcel src/pages/index.pug & node server.js",
+    "dev": "parcel src/pages/index.pug -p 3000 --no-cache",
+    "build": "parcel src/pages/index.pug -p 3000 --no-cache"
```

`parcel src/pages/index.pug` запускает дев сборку. Правильней будет
```suggestion
    "start": "npm run build & node server.js",
    "dev": "parcel src/pages/index.pug -p 3000 --no-cache",
    "build": "parcel build src/pages/index.pug"
```